### ISSUE TITLE
Add babelTask to just-scripts

### ISSUE THREAD

**Issue by Thisen** [microsoft/just#308]
At my company, we use just-scripts for tooling around our monorepo.

We want to introduce Babel to our TypeScript for the speed and plugins (Like react-refresh).

Therefore I propose adding a `babelTask` to `just-scripts`.

I'm up for developing this myself.

**Comment by jkovzels** [comment]
> Issues seem to have gone stale.

this bot does not make any sense and and no information. But it is good cluttering my inbox with this non-sense.

**Comment by ecraig12345** [comment]
I agree, this is a valid request which shouldn't have been marked for closure.

**Comment by ecraig12345** [comment]
@kenotron This will also be important for fabric/fluent soon.
